### PR TITLE
Added MeTube Template

### PR DESCRIPTION
**Pull request by Ponkhy**

### CODE REVIEW

File: templates/metube.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        },
+        proxy: {
+          port: 8081,
+          secure: true,
+        },
+        domains: [{ name: domain }],
+        volumes: [
+          {
+            type: "volume",
+            source: downloadPath,
```
Comment (deiucanta) [inline templates/metube.ts:62]: I think we should not make this configurable. It should be "downloads" by default.

Comment (Ponkhy) [reply templates/metube.ts:62]: Normally i would agree, but in this case it's pretty likely that users want to change it and at the end this is easier than explaining how to create a volume for this image afterwards.

Comment (deiucanta) [reply templates/metube.ts:62]: Makes sense. Then let's name it "Downloads Volume Name" (maybe). Or, if you want to keep it as a path, then the volume type should be `bind` not `volume`

Comment (deiucanta) [reply templates/metube.ts:62]: @Ponkhy I want to merge this PR. Can you please change "Download Path" to "Downloads Volume Name"?

Comment (Ponkhy) [reply templates/metube.ts:62]: Sorry for the delay, it's renamed now.

**Pull request merged**